tests/network-ovn

```diff
@@ -2019,6 +2019,94 @@ ovn_l3only_tests() {
     lxc network delete lxdbr0
 }
 
+# countDynamicLeases is a helper function that counts dynamic leases of the instance within
+# all network leases.
+countDynamicLeases(){
+        network="$1"
+        instance="$2"
+        count=$(lxc network list-leases "${network}" --format compact | awk -v inst="${instance}" '$1 == inst && $4 == "DYNAMIC" { count++ } END { print count+0 }')
```

Comment:
I would advise to use `--format csv` as `compact` sometimes omit some columns:

```
# Shows only 3 columns
$ lxc network list-leases lxdbr0 --format compact | grep -wF lxdbr0.gw
  lxdbr0.gw                       10.76.66.1                              GATEWAY  
  lxdbr0.gw                       fd42:b2a9:e8fd:1156::1                  GATEWAY

# Shows 4 columns:
$ lxc network list-leases lxdbr0 --format compact | grep -wF t1
  t1           00:16:3e:cb:e2:d4  fd42:b2a9:e8fd:1156:216:3eff:fecb:e2d4  DYNAMIC
```

With `csv` format, you can then use `awk -F,` to have it split on `,`.